unumpy/_multimethods.py

```diff
@@ -826,11 +824,15 @@ def broadcast_to(array, shape, subok=False):
     return (array,)
 
 
+def _normalize_axis(ndim, axis):
+    return [ndim + ax if ax < 0 else ax for ax in axis]
```

Comment:
You should also move the `Axis is out of bounds` error into here.

Reply:
Maybe include both `Axis is out of bounds` and `Repeated axis` errors then?

Reply:
@joaosferreira I'd add a `allow_repeats=False` kwarg, as some functions allow repeats. I cannot think of any off the top of my head though.

Reply:
I'd also handle scalar `axis`.

Reply:
I've pushed the changes for `_normalize_axis`.